snmp_cisco/README.md

```diff
@@ -0,0 +1,29 @@
+# Agent Check: snmp_cisco
+
+## Overview
+
+Cisco is the worldwide leader in IT, networking, and cybersecurity solutions. Install the integration to monitor all of your Cisco appliances, including routers, switches, voice gear, security appliances and more.
+
+Collect SNMP metrics from Cisco appliances, including:
+
+- Cisco Catalyst
+- Cisco Adaptive Security Appliance (ASA)
+- Cisco Meraki (note: additional events can be collected from Meraki through the [Meraki integration tile][1])
+- Cisco Nexus
+- Cisco ICM
+- Cisco ISR
+- Cisco UC Virtual Machines
+
+## Setup
+
+To install and configure the SNMP integration, see the [Network Device Monitoring][2] documentation.
+
```

Comment:
This readme is missing the Metrics, events, and service check sections, is that intentional?

Reply:
Yes, because there is none. 

We could also mention it:

```
### Service Checks

snmp-cisco does not include any service checks.

### Events

snmp-cisco does not include any events.
```